src/index.html

```diff
@@ -15,8 +15,29 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/"class="logo__img"><img src="./images/logo.png"alt="Moyo Logo"/></a>
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="/apple"class="nav__link is-active">Apple</a>
+            <a href="/samsung"class="nav__link">Samsung</a>
+            <a href="/smartphones"class="nav__link">Smartphones</a>
+            <a href="/laptops & computers"class="nav__link"
+              data-qa="hover">Laptops & Computers</a>
+            <a href="/gadgets"class="nav__link">Gadgets</a>
+            <a href="/tablets"class="nav__link">Tablets</a>
+            <a href="/photo"class="nav__link">Photo</a>
+            <a href="/video"class="nav__link">Video</a>
```

Comment:
Don't use spaces in the `href` attribute of the `<a>` tags. Also, the `href` attribute should start with `#` if it's meant to be a placeholder or an anchor link on the same page.